custom/enikshay/nikshay_datamigration/models.py

```diff
@@ -129,6 +136,19 @@ def patient_type_choice(self):
         }[self.Ptype]
 
     @property
+    def occupation(self):
+        return {
+            '4': 'engineer',
+            '5': 'doctor_',
```

Comment:
oh, I just copied that from https://docs.google.com/document/d/11QAaFMy97XRaXl_dNEwIv74ksJX3DlUpvtyRtuxSaBo/edit#heading=h.n7po03tgw7w4

should it be deleted?